front/src/components/App/MaintenanceArea/AuctionSitesList.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import {deleteAucitonSite} from "./Utils";
+import {AuctionSite} from "./AuctionSitesManagement";
+import {AuctionSitesInterface} from './AuctionSitesAddition'
+
+const AuctionSitesList = (auctionInterface: AuctionSitesInterface) => {
```

Comment:
jeśli nazywamy propsy jako całość (bez destrukturyzacji) to niech nazywają się props. W dodatku ten interfejs jest wykorzystywany jeszcze gdzieś indziej jako propsy wiec coś zawiodło skoro 2 komponenty wykonujące inne zadania wykorzystują zestaw tych samych propsów (dość złożonych i specyficznych)